crates/store/src/db/models/queries/accounts.rs

```diff
@@ -38,38 +37,35 @@ use miden_objects::account::{
 };
 use miden_objects::asset::{Asset, AssetVault, AssetVaultKey, FungibleAsset};
 use miden_objects::block::{BlockAccountUpdate, BlockNumber};
-use miden_objects::{Felt, Word};
+use miden_objects::{Felt, FieldElement, Word};
 
 use crate::db::models::conv::{SqlTypeConvert, nonce_to_raw_sql, raw_sql_to_nonce};
 use crate::db::models::{serialize_vec, vec_raw_try_into};
 use crate::db::{AccountVaultValue, schema};
 use crate::errors::DatabaseError;
 
 type StorageMapValueRow = (i64, String, Vec<u8>, Vec<u8>);
-
-/// Select the latest account details by account id from the DB using the given
```

Comment:
I think this was probably deleted by mistake. In general, I'd move the content (slightly re-phrased) from lines 56 - 58 to here.